jamf-upload.sh

```diff
@@ -199,10 +199,12 @@ processors_directory="$DIR/JamfUploaderProcessors"
 ## ARGUMENTS ##
 ###############
 
-echo "" > "$temp_processor_plist"  # ensure an empty processor at the start of the run
+rm -rf "$temp_processor_plist" # delete any existing file (or folder) at temp_processor_plist path
+/usr/libexec/PlistBuddy -c 'Clear dict' "$temp_processor_plist" # ensure an empty processor at the start of the run
+# NOTE: DO NOT use "plutil -create" to create a new empty plist since that option is only available on macOS 12 Monterey and newer.
```

Comment:
Are there any other `plutil` commands you have used that have a minimum macOS that I should be aware of?

Reply:
All the other "plutil -replace" commands support at least 10.11 when that option was added to "plutil" (https://scriptingosx.com/2016/11/editing-property-lists/).